app/scripts/angular-apimock.js

```diff
@@ -247,15 +264,13 @@ angular.module('apiMock', [])
 		}
 
 		function removeFallback(res) {
-			var found = false;
-			angular.forEach(fallbacks, function (fallback, index) {
-				if (fallback.method === res.method && fallback.url === res.url) {
-					found = true;
-					fallbacks.splice(index, 1);
-				}
-			});
+			var startLength = fallbacks.length;
+			fallbacks = $filter('filter')(fallbacks, {
+				method: '!' + res.method,
+				url: '!' + res.url
+			}, true);
 
-			return found;
+			return startLength > fallbacks.length;
```

Comment:
Thanks! I feel it's super obscure to the people that don't know how it should be called. Maybe I should link to https://docs.angularjs.org/api/ng/filter/filter ?

But it feels like it's a lot better tested anyway and I was loosing code coverage because I wasn't testing it enough, which feels unnecessary since the point of apiMock isn't to reinvent "remove elements from array".